gpiozero/boards.py

```diff
@@ -1324,6 +1324,105 @@ def __init__(self, *, pwm=False, initial_value=False, pin_factory=None):
         )
 
 
+class ModMyPiStar(LEDBoard):
+    """
+    Extends :class:`LEDBoard` for the `ModMyPi Programmable Christmas Tree Star`_:
+    a Christmas tree star with 30 white LEDs.
+
+    The 30 white LEDs can be accessed through the attributes inner and outer.
+    The inner attribute controls the five inner LEDs as a single group, while
+    the outer attribute controls the 25 outer LEDs. Those 25 LEDs can be controlled
+    individually.
```

Comment:
Thanks, this looks much better now! Only a small tweak, but perhaps "Those 25 LEDs can be controlled individually." would read better as "The 25 outer LEDs can also be controlled individually." ?